src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -1382,16 +1356,8 @@ def disable_features(cmd, client, resource_group_name, cluster_name, kube_config
     }
     telemetry.add_extension_event('connectedk8s', kubernetes_properties)
 
-    # Uncomment below custom locations related params are available
-
-    # if disable_cluster_connect:
-    #     helm_values = get_all_helm_values(client, cluster_name, resource_group_name, configuration, kube_config, kube_context)
-    #     # helm_values_flattened = utils.flatten(helm_values)
-    #     if helm_values.get('systemDefaultValues').get('customlocations').get('enabled') == True: # update this line
```

Comment:
Dont remove these, this is intended. We have to check if the existing helm parameter has customlocations enabled or not. Uncomment

Reply:
Here, I had done everything from displaying user message, checking helm parameter, everything. Just uncomment and verify.

Reply:
Yeah, this was required. I have added it back.